### ISSUE TITLE
Add support for document_version to run_command block for aws_ssm_maintenance_window_task

### ISSUE THREAD

**Issue by rafmagns-skepa-dreag** [hashicorp/terraform-provider-aws#13700]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Support was added for `document_version` in `automation_parameters` blocks a while back. Support is also already included for `document_hash` and `document_hash_type` in `run_command` blocks. I would like to request support for `document_version` inside `run_command` blocks. It is supported in the aws-sdk and the [command line](https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_MaintenanceWindowRunCommandParameters.html#systemsmanager-Type-MaintenanceWindowRunCommandParameters-DocumentVersion).

### New or Affected Resource(s)

* aws_ssm_maintenance_window_task

### Potential Terraform Configuration

```hcl
resource aws_ssm_maintenance_window_task "ecs_patch_task_scan" {
  name             = "ecs-patch-task"
  window_id        = aws_ssm_maintenance_window.ecs_window.id
  max_concurrency  = 1
  max_errors       = 1
  priority         = 1
  task_type        = "RUN_COMMAND"
  task_arn         = "AWS-RunPatchBaseline"
  service_role_arn = data.aws_iam_role.SSM-Role.arn

  task_invocation_parameters {
    run_command_parameters {
      document_version = "$LATEST"
      parameter {
        name   = "Operation"
        values = ["Scan"]
      }
      parameter {
        name   = "RebootOption"
        values = ["NoReboot"]
      }
    }
  }

  targets {
    key    = "WindowTargetIds"
    values = [aws_ssm_maintenance_window_target.ecs_target.id]
  }
}
```

### References

* #6431
* #5762

**Comment by rafmagns-skepa-dreag** [comment]
I would like to work on this issue

**Comment by bflad** [comment]
This functionality has been merged and will release with version 3.28.0 of the Terraform AWS Provider, later today.
